languages/js/test/parity.ts

```diff
@@ -71,6 +81,8 @@ if (
     oso.queryRule('testMethodCalls', new A('hello'), new B.C('hello')).next()
       .done,
     oso.queryRule('testOr').next().done,
+    oso.queryRule('testUnifyClass', A).next().done,
+    oso.queryRule('testDerefJava').next().done,
```

Comment:
That was a happy accident.
How about `testDerefJavas`?

Reply:
These are a little unfortunate in that it's supposed to be a parity test, but the languages don't agree on a common method for this.

Reply:
For the other parity tests that required language specific stuff, I just loaded each rule string separately in the language-specific file instead of adding it to the shared polar file